plugins/html/help.md

```diff
@@ -63,218 +57,221 @@ Example output:
-
+| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
+|doc|string|None|True|Document to transform|None|<!DOCTYPE html><html><head><title>Rapid7 InsightConnect</title></head><body><p>Automate with InsightConnect!</p></body></html>|
+|remove_scripts|boolean|None|False|Remove non-HTML scripts from the document|None|False|
+  
 Example input:
 
 ```
 {
-  "doc": "\u003c!DOCTYPE html\u003e\u003chtml\u003e\u003chead\u003e\u003ctitle\u003eRapid7 InsightConnect\u003c/title\u003e\u003c/head\u003e\u003cbody\u003e\u003cp\u003eConvert HTML to EPUB\u003c/p\u003e\u003c/body\u003e\u003c/html\u003e"
+  "doc": "<!DOCTYPE html><html><head><title>Rapid7 InsightConnect</title></head><body><p>Automate with InsightConnect!</p></body></html>",
+  "remove_scripts": false
 }
 ```
 
 ##### Output
 
 |Name|Type|Required|Description|Example|
-|----|----|--------|-----------|-------|
-|epub|bytes|False|Epub file|UEsDBBQAAggAAPAe91BvYassFAAAABQAAAA...|
-
+| :--- | :--- | :--- | :--- | :--- |
+|text|string|False|String without HTML tags|Automate with InsightConnect!|
+  
 Example output:
 
 ```
 {
-  "epub": "UEsDBBQAAggAAPAe91BvYassFAAAABQAAAA..."
+  "text": "Automate with InsightConnect!"
 }
 ```
 
-#### Text
-
-This action is used to strip an HTML string of all tags and return only the text.
+#### Validate
+  
+This action is used to validate an HTML document using the [W3 validator](https://validator.w3.org)
 
 ##### Input
 
 |Name|Type|Default|Required|Description|Enum|Example|
-|----|----|-------|--------|-----------|----|-------|
-|doc|string|None|True|Document to transform|None|<!DOCTYPE html><html><head><title>Rapid7 InsightConnect</title></head><body><p>Automate with InsightConnect!</p></body></html>|
-|remove_scripts|boolean|None|False|Remove non-HTML scripts from the document|None|False|
-
+| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
+|html_contents|string|None|True|HTML Contents|None|<!DOCTYPE html><html><head><title>Rapid7 InsightConnect</title></head><body><p>Automate with InsightConnect!</p></body></html>|
+  
 Example input:
 
 ```
 {
-  "doc": "\u003c!DOCTYPE html\u003e\u003chtml\u003e\u003chead\u003e\u003ctitle\u003eRapid7 InsightConnect\u003c/title\u003e\u003c/head\u003e\u003cbody\u003e\u003cp\u003eAutomate with InsightConnect!\u003c/p\u003e\u003c/body\u003e\u003c/html\u003e",
-  "remove_scripts": false
+  "html_contents": "<!DOCTYPE html><html><head><title>Rapid7 InsightConnect</title></head><body><p>Automate with InsightConnect!</p></body></html>"
 }
 ```
 
 ##### Output
 
 |Name|Type|Required|Description|Example|
-|----|----|--------|-----------|-------|
-|text|string|False|String without HTML tags|Automate with InsightConnect!|
-
+| :--- | :--- | :--- | :--- | :--- |
+|validated|boolean|False|HTML Syntax Validation Status|True|
+  
 Example output:
 
 ```
 {
-  "text": "Automate with InsightConnect!"
+  "validated": true
 }
 ```
-
 ### Triggers
+  
+*This plugin does not contain any triggers.*
+### Tasks
```

Comment:
I think what Tomas meant here was add a newline between the `Tasks` title and the info above like so
```suggestion
*This plugin does not contain any triggers.*

### Tasks
```